README.md

```diff
@@ -104,10 +104,13 @@ pip install datatable
 On Linux you can install a binary distribution as
 ```sh
 # If you have Python 3.5
-pip install https://s3.amazonaws.com/h2o-release/datatable/stable/datatable-0.9.0/datatable-0.9.0-cp35-cp35m-linux_x86_64.whl
+pip install https://s3.amazonaws.com/h2o-release/datatable/stable/datatable-0.10.0/datatable-0.10.0-cp35-cp35m-linux_x86_64.whl
 
 # If you have Python 3.6
-pip install https://s3.amazonaws.com/h2o-release/datatable/stable/datatable-0.9.0/datatable-0.9.0-cp36-cp36m-linux_x86_64.whl
+pip install https://s3.amazonaws.com/h2o-release/datatable/stable/datatable-0.10.0/datatable-0.10.0-cp36-cp36m-linux_x86_64.whl
+
+# If you have Python 3.7
+pip install https://s3.amazonaws.com/h2o-release/datatable/stable/datatable-0.10.0/datatable-0.10.0-cp37-cp37m-linux_x86_64.whl
```

Comment:
Can't we just do `pip install datatable` here?

Reply:
we could, but then there wouldn't be any place left where we mention the S3 repository, so how would anybody be able to find those links?

Reply:
May be we add a link to `https://s3.amazonaws.com/h2o-release/datatable/stable/` in the docs somewhere? Could be a section called `Customized wheels` or something like that.

Reply:
I'm not opposed to it, there are many ways in which our documentation can be improved.

Reply:
But we should defer this task to after the release

Reply:
Sure, that's not a big deal...